tortoise/tortoise_test.go

```diff
@@ -1480,7 +1480,7 @@ func ensureBaseAndExceptionsFromLayer(tb testing.TB, lid types.LayerID, votes *t
 }
 
 func TestBaseBallotEvictedBlock(t *testing.T) {
-	const size = 10
+	const size = 12
```

Comment:
i don't know why test fails. changing this to 12 stabilized the test. maybe the random-ness is altered again.

Reply:
i will address random weights in one of the open prs